### PR TITLE
fix household calculation

### PR DESCRIPTION
**Pull request by Rohit25negi**
Earlier no date check was applied to house hold count. it was just counting the households for all the rows in the ucr. Which was leading to also include cases which are opened in next month.
This code change checks to only include the household which are opened before the end of current month.
ticket: https://dimagi-dev.atlassian.net/browse/ICDS-510
PS: this will need historical data correction also.

### CODE REVIEW

File: custom/icds_reports/utils/aggregation_helpers/monolith/agg_awc.py [hunk 1]
```diff
@@ -230,13 +230,14 @@ def updates(self):
                 owner_id,
                 sum(open_count) AS cases_household
             FROM "{household_cases}"
+            WHERE opened_on<= %(end_date)s
```
Comment (Rohit25negi) [inline custom/icds_reports/utils/aggregation_helpers/monolith/agg_awc.py]: Should I make this same change to citus also?

**Comment by Rohit25negi** [comment]
output Test cases changed because fixture is changed.
the actual reason of decrease is number is because 13 household's opened_on date is shifted to 2017-05-01 month therefore they are not counted in 2017-04-01 but counted in 2017-05-01.

Which is the validation test for the code change.

File: custom/icds_reports/utils/aggregation_helpers/monolith/agg_awc.py (hunk 1 above)
Comment (snopoke) [reply custom/icds_reports/utils/aggregation_helpers/monolith/agg_awc.py]: yes please. You'll also need to run `manage.py update_icds_aggregation_output`

**Review by calellowitz — APPROVED** [review]
Is the fact that some numbers went down because you purposefully added open dates after the aggregation to the test fixture? If not, it would be good to add those. But otherwise looks great

**Comment by Rohit25negi** [comment]
> Is the fact that some numbers went down because you purposefully added open dates after the aggregation to the test fixture? If not, it would be good to add those. But otherwise looks great

Hey, Yes it was intentional. to add the test data for change I made

File: custom/icds_reports/utils/aggregation_helpers/monolith/agg_awc.py (hunk 1 above)
Comment (Rohit25negi) [reply custom/icds_reports/utils/aggregation_helpers/monolith/agg_awc.py]: DOne

**Comment by Rohit25negi** [comment]
@snopoke, added change to distributed script also.

**Review by snopoke — APPROVED** [review]

**Pull request merged**